pkg/crc/machine/generate_bundle_linux.go

```diff
@@ -0,0 +1,50 @@
+package machine
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/machine/bundle"
+	crcos "github.com/code-ready/crc/pkg/os"
+)
+
+func copyDiskImage(destDir string, crcBundleMetadata *bundle.CrcBundleInfo) error {
+	tmpDir := filepath.Join(constants.MachineCacheDir, "image-rebase")
+	_ = os.RemoveAll(tmpDir) // clean up before using it
```

Comment:
If I somehow create a `~/.crc/cache/image-rebase` -> `~` symlink, this will wipe my homedir...

Reply:
This is not the first place where I am using this logic, it is already part of our codebase https://github.com/code-ready/crc/blob/master/pkg/crc/machine/bundle/repository.go#L114-L118 . The reason I am using `~` because `image-rebase` takes a lot of space and `/tmp` might not have that much but user home directory might have.

Reply:
> This is not the first place where I am using this logic, it is already part of our codebase https://github.com/code-ready/crc/blob/master/pkg/crc/machine/bundle/repository.go#L114-L118 

We can try to do better in this PR rather than adding one more use of this problematic pattern?

Reply:
Actually, RemoveAll does not follow symlink to dirs, it only removes the symlink. I tested it after seeing its code:
https://golang.org/src/os/removeall_at.go
```golang
	// Simple case: if Remove works, we're done.
	err := Remove(path)
	if err == nil || IsNotExist(err) {
		return nil
	}
```